gnomad_qc/v4/variant_qc/random_forest.py

```diff
@@ -80,7 +79,8 @@ def train_rf(
         Default is False.
     :param sibling_singletons: Whether to use sibling singletons for training. Default
         is False.
-    :param adj: True if using _adj genotypes for singletons. If False, using _raw.
+    :param adj: Whether to use adj genotypes for transmitted/sibling singletons instead
+        of raw. Default is False.
```

Comment:
```suggestion
        of raw. Default is False and raw is used. 
```
Unless this is ugly or wouldn't pass style/linting checks